Main.py

```diff
@@ -225,7 +225,7 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
     outfilebase = 'AP_' + multiworld.seed_name
 
     output = tempfile.TemporaryDirectory()
-    with output as temp_dir:
+    with (output as temp_dir):
```

Comment:
Why parens?

Reply:
Yeah, I think this looks more confusing with the parentheses.
It's not normal for `with` statements.

`as` is not a binary operator acting on `output` and `temp_dir`
The parentheses make it look like that's what it is.